crates/controller/src/hud.rs

```diff
@@ -79,3 +89,30 @@ fn process_events(
         }
     }
 }
+
+pub fn spawn_hud(
+    mut commands: Commands,
+) {
+    commands.spawn(
```

Comment:
Is this needed? I think that setting absolute position to all three children and not using the parent would be more concise.